addons/website_blog/views/snippets/s_blog_posts.xml

```diff
@@ -16,8 +16,8 @@
         <a class="s_blog_posts_post_cover flex-grow-0 flex-shrink-0 align-self-baseline position-relative" t-att-href="data['call_to_action_url']" t-att-title="'Read' + data['name']">
             <t t-call="website.record_cover">
                 <t t-set="_record" t-value="record"/>
-                <t t-set="_resize_height" t-value="128"/>
-                <t t-set="_resize_width" t-value="128"/>
```

Comment:
It works but I think you should fix that in JS instead because we can't modify the XML templates in stable (and the existing DBs won't have the fix)  :wink:

Reply:
It wouldn't be that simple, because it's part of the server side rendering, which ultimately calls a private model method `_get_background`.
We can't just replace part of the URL in JS, we would have to redo the method call to ensure having a correct URL.
It would ultimately also mean loading twice the image, which wouldn't be ideal bandwidth wise.

This is probably one of those cases where it's not fixing a TB but just improving things that went under the radar for long time, for which case it's quite fine to require a module update to benefit from the fix or fine if only helping new DBs.

Reply:
I mean modifying the template in JS, like patching it to replace the numbers so it would not be done in the XML template  :slightly_smiling_face: 
Is it not possible ?

Reply:
No since it's a server side view, everything is already done/rendered once it reaches the browser/client.
Unless I am misunderstanding what you mean?

Reply:
Ah indeed, I forgot it was the dynamic snippet and not a simple option template :grimacing: 
I looked around and there doesn't seem to be an easy way to patch it so it is okay like this :slightly_smiling_face: